src/dashboard/apps/consent/views.py

```diff
@@ -219,3 +227,52 @@ def _build_consent_object(
             **form_values_copy,
             signature_location=settings.CONSENT_SIGNATURE_LOCATION,
         )
+
+    def _send_email(self) -> None:
+        """Send an email to the user for consent validation.
+
+        This method constructs and sends an email using the Anymail library based on
+        predefined configurations.
+        It logs errors with Sentry and fails silently in case of improper configurations
+        or API errors.
+        The email contains user information and a link to return in the dashboard.
+
+        Raises:
+            ImproperlyConfigured: If 'template_id' is missing in the email configuration
+            AnymailRequestsAPIError: If an error occurs while sending the email.
+        """
+        user = self.request.user
+        email_config = settings.DASHBOARD_EMAIL_CONFIGS["consent_validation"]
+
+        # fail silently and send a sentry log
+        if not email_config.get("template_id"):
+            sentry_sdk.capture_exception(
+                ImproperlyConfigured(
+                    "The 'template_id' parameter is missing in the configuration "
+                    "(DASHBOARD_EMAIL_CONFIGS['consent_validation'])."
+                )
+            )
```

Comment:
Can it be check as early as possible, e.g. when the application is bootstrapping?

Reply:
Finally there is no need to check, `django.environ` takes care of it for us!
if the param is `null` while we want to retrieve it via an `env.int()`, a `ValueError` is raised during the application bootstrap :)